net-nds/389-ds-base/389-ds-base-1.4.2.15.ebuild

```diff
@@ -1,39 +1,40 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-WANT_AUTOMAKE="1.13"
+WANT_AUTOMAKE="1.16"
 
-inherit user eutils multilib flag-o-matic autotools
-
-DESCRIPTION="389 Directory Server (core librares and daemons )"
-HOMEPAGE="http://www.port389.org/"
-SRC_URI="http://www.port389.org/sources/${P}.tar.bz2"
+inherit eutils multilib flag-o-matic autotools systemd db-use
 
+DESCRIPTION="389 Directory Server (core librares and daemons)"
+HOMEPAGE="https://directory.fedoraproject.org/"
+MY_P="${PN}-${PN}-${PV}"
+SRC_URI="https://pagure.io/389-ds-base/archive/${PN}-${PV}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
 LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="autobind auto-dn-suffix debug doc +pam-passthru +dna +ldapi +bitwise presence kerberos selinux"
+IUSE="autobind auto-dn-suffix debug doc +pam-passthru +dna +ldapi +bitwise presence selinux systemd"
 
-# Pinned to db:4.8 as it is the current stable, can change to a later db version < 6 when they stabilize.
-# The --with-db-inc line in econf will need to be updated as well when changing db version.
 COMMON_DEPEND="
-	sys-libs/db:4.8
+	>=sys-libs/db-4.2:=
 	>=dev-libs/cyrus-sasl-2.1.19
 	>=net-analyzer/net-snmp-5.1.2
 	>=dev-libs/icu-3.4:=
 	>=dev-libs/nss-3.22[utils]
 	dev-libs/nspr
-	>=dev-libs/svrcore-4.1.2
```

Comment:
this should be blocked, not removed, at least for a while

Reply:
I agree, will look into it